embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/PublicApiTest.kt

```diff
@@ -136,4 +139,23 @@ internal class PublicApiTest {
             }
         }
     }
+
+    @Test
+    fun `static logger and SDK logger instance log to the same internal error service`() {
```

Comment:
This will be deleted later on with further refactoring to eliminate having two loggers user the same internal error service, but I left this here in case the PRs aren't merged together.